### ISSUE TITLE
I validated all my tasks, but lost health and didn't get my half-your-level bonus

### ISSUE THREAD

**Issue by Avistew** [HabitRPG/habitica#3911]
I finished all my daily tasks yesterday and validated them before 11pm. I left HabitRPG open in a tab as I do all day. I checked it right after midnight, and I had lost 10 HP and didn't get my bonus for doing all my dailies.

I can't find a place to contact people so they can give me my health and bonus back except here. It would be more practical if you had an email address for this kind of thing as it's not quite a bug, but still pretty annoying. I lost my streaks and it makes me want to just give up altogether. Please fix it ASAP.

**Comment by lefnire** [comment]
I am so sorry that this happened to you :( That's incredibly frustrating,
especially since it wasn't at all your fault. There's good news, though -
there is a way that you can restore everything!

To restore your gold, items, level, and XP, you can go to Settings > Site >
Fix Character Values. Unfortunately we can't check what your exact values
were before they were lost, but it's okay to round up to account for all of
the stress of this experience :(

To restore your streaks, go to Task Edit and scroll down to where it says
"Restore Streak." You'll be able to enter the number manually. If you're
not sure what your streak was, you can check the graphs to approximate.

For the future, you may be interested in the Data Display Tool, found here:
https://oldgods.net/habitrpg/habitrpg_user_data_display.html If you find
that HabitRPG is having trouble for whatever reason, you can use this tool
to check how many of your Dailies are completed, to prevent this same thing
from happening again.

We're very sorry about all of this. Hopefully these solutions will help
you. We really do appreciate your donations to the site, and hope that you
have a better experience in the future!

**Comment by Avistew** [comment]
Thanks a lot for your answer! I was really worried when I lost my streaks
that I would not keep going, as I have in the past completely given up
after a missed day even if I had been doing something for months! (That was
before I used HabitRPG though).

I'm happy to say that HabitRPG provides enough other incentives that I kept
going, so my main worry is gone and I'm sorry I overreacted.

I'm very glad that there are ways to fix the issues, thank you for letting
me know! I will use the Data Display Tool from now on.

2014-08-25 10:38 GMT-07:00 Tyler Renelle notifications@github.com:
